webapp/src/js/controllers/contacts.js

```diff
@@ -361,6 +361,10 @@ var _ = require('underscore'),
     };
 
     var canViewLastVisitedDate = function() {
+      if (Session.isDbAdmin()) {
```

Comment:
Are admins disabled because they have too much data? If so we also need to block "national_admins" by using `Session.isAdmin`. I might go even further and use `Session.isOnlineOnly` because online users will probably all have lots of data.

Reply:
Ahh right, that makes sense, and db admin is the right perm to check. Thanks for explaining.